src/pages/account-restore.tsx

```diff
@@ -121,9 +124,9 @@ export const AccountRestore = (_: Props) => {
 						<p class="username">{profile?.username}</p>
 					</div>
 					<div class="btns">
-						<a class="btn btn-light" href={ACCOUNT}>
+						<Link className="btn btn-light" to={ACCOUNT}>
```

Comment:
I am a bit confused, don't you want to use `FlexLink` here and in other places instead of `Link`?

Reply:
The idea of `FlexLink` is mostly to be used in more general components. For example, the button could be an external link or an internal one. But in pages we know where we want to go next, so I feel like it's cleaner / clearer to use the right tag from the get-go, i.e. `a` or `Link`.

To be honest I'm not even really sure if it's needed, as the button components most likely always redirect internally.